website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,189 @@
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/prettier.gif" width="95%" title="Format YAML, Markdown, and JSON files using Prettier."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-sql-popup.jpg" width="95%" title="Use the Config button to select your tool."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-sqlfluff-config.jpg" width="95%" title="Customize linting by configuring your own linting code rules, including dbtonic linting/styling."/>
+
+</DocCarousel>
+
+## Lint
+
+With the dbt Cloud IDE, you can seamlessly use [SQLFluff](https://sqlfluff.com/), a configurable SQL linter, to warn you of complex functions, syntax, formatting, and compilation errors. This integration allows you to run checks, fix, and display any code errors directly within the Cloud IDE:
+
+- Works with Jinja and SQL, 
+- Comes with built-in [linting rules](https://docs.sqlfluff.com/en/stable/rules.html). You can also [customize](#customize-linting) your own linting rules.
+- Empowers you to [enable linting](#enable-linting) with options like **Lint** (displays linting errors and recommends actions) or **Fix** (auto-fixes errors in the IDE).
+- Displays a **Code Quality** tab to view code errors, and provides code quality visibility and management. 
+
+### Enable linting
+
+1. To enable linting, make sure you've opened a `.sql` file.
+2. Click on the **`</> Config`** button on the right side of the console. 
+3. In the code quality tool config pop up, you have the option to select **sqlfluff** or **sqlfmt**. 
+4. To lint your code, select the **sqlfluff** radio button. (Use sqlfmt to [format](#format) your code)
+5. Once you've selected the **sqlfluff** radio button, go to the console section (located below the **File editor**) to select the **Lint** or **Fix** dropdown button:
+    - The **Lint** button displays linting issues in the IDE as wavy underlines in the **File editor**. You can hover over an underlined issue to display the details and actions, including a **Quick Fix** option to fix all or specific issues. After linting, you'll see a message confirming the outcome. Linting doesn't rerun after saving. Click on the **Lint** button again to rerun linting.
+    - The **Fix** dropdown button auto-fixes errors in the **File editor**. Once you've auto-fixed, you'll see a message confirming the outcome. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-lint-format-console.gif" width="95%" title="Use the Lint or Fix button in the console section to lint or auto-fix your code."/>
+
+### Customize linting
+
+SQLFluff is a configurable SQL linter, which means you can configure your own linting rules instead of using the default linting settings in the IDE.  
+
+To configure your own linting rules:
+
+1. Create a new file in the root project directory (the parent or top-level directory for your files).
+2. Name the file `.sqlfluff` (make sure you add the `.` before `sqlfluff`).
+3. Add your custom config code. 
```

Comment:
Is there a link we can provide that outlines how to create config code?

Reply:
i think the best reliable source is prob the sqlfluff page has that info here: https://docs.sqlfluff.com/en/stable/configuration.html#new-project-configuration

confirming with @dave-connors-3 if this is the best source?

Reply:
i dont want to provide non official type links since it's not the main scope of the page

Reply:
dave you da man